autode/calculation.py

```diff
@@ -37,7 +39,13 @@ def get_solvent_name(molecule, method):
         logger.info('Calculation is in the gas phase')
         return None
 
-    solvent_name = getattr(molecule.solvent, method.name)
+    if type(molecule.solvent) is str:
+        # Solvent could be a string from e.g. cgbind
+        solvent_name = molecule.solvent
```

Comment:
if molecule.solvent is a string should we check its available?

Reply:
Ideally, but I didn't know how to implement it..